pallets/committee/src/lib.rs

```diff
@@ -431,6 +432,31 @@ pub mod pallet {
 			Self::deposit_event(Event::NewConstituent(constituent));
 			Ok(())
 		}
+
+		/// Remove council or constituent via governance
+		///
+		/// This call can only be called after the approval of the committee
+		#[pallet::weight((T::WeightInfo::remove_member(), DispatchClass::Operational))]
```

Comment:
Also not required, right?
```suggestion
		#[pallet::weight(T::WeightInfo::remove_member())]
```